src/main/java/com/sericulture/authentication/controller/UserInfoController.java

```diff
@@ -58,11 +58,11 @@ public AuthApiResponse checkToken() {
         return authApiResponse;
     }
 
-    @PostMapping("/register")
+  /*  @PostMapping("/register")
     public AuthApiResponse registerUser(@RequestBody UserInfo userInfo) {
```

Comment:
Is registration centralised from master data? If yes we can remove this code instead of commenting!